getting_started/dev-environment/macOS.md

```diff
@@ -0,0 +1,73 @@
+# Instructions for macOS
+
+Install [Homebrew](http://brew.sh/) to give your mac a package manager. Open up your terminal emulator of choice and run:
+
+```shell
+$ /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+```
+
+Install some basic command line tools by running:
+
+```shell
+$ brew install coreutils
+```
+
+Install git:
+
+```shell
+$ brew install git
+```
+
+Install python and its package manager, pip:
+
+```shell
+$ brew install python
+$ curl -O http://python-distribute.org/distribute_setup.py
+$ python distribute_setup.py
+$ curl -O https://raw.github.com/pypa/pip/master/contrib/get-pip.py
+$ python get-pip.py
+$ rm distribute_setup.py get-pip.py
+```
+
+Install virtualenv and virtualenvwrapper:
+
+```shell
+$ pip install virtualenv virtualenvwrapper
+```
+
+Add the following to your `~/.bash_profile`:
+
+```shell
+# Create a backup of your .bash_profile
+cp ~/.bash_profile ~/.bash_profile-org
+
+# Be careful with this command
+printf '\n%s\n%s\n%s' '# virtualenv' 'export WORKON_HOME=~/virtualenvs' \
+'source /usr/local/bin/virtualenvwrapper.sh' >> ~/.bash_profile
+```
+
+Enable virtualenvwrapper:
+
+```shell
+$ source ~/.bash_profile
+```
+
+Install virtualbox and vagrant for testing changes locally:
+
+* Visit the [VirtualBox Downloads page](https://www.virtualbox.org/wiki/Downloads) and click on `OS X Hosts`. A `.dmg` will begin to download.
+* Navigate to your downloads directory in Finder and double-click on the `.dmg` file. Follow the on-screen instructions to complete the installation.
+* Visit the [Vagrant Downloads page](https://www.vagrantup.com/downloads.html) and click on `Universal (32 and 64-bit)` under the `Mac OS X` header. Another `.dmg` file will begin to download.
+* Navigate to your downloads directory in Finder and double-click on the `.dmg` file. Follow the on-screen instructions to complete the installation.
```

Comment:
This block can be replaced with a simple link to the installation pages for the software. Maybe at the top of the file with brew?
```markdown
Install required software:
- [Homebrew](http://brew.sh/)
- [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
- [Vagrant](https://www.vagrantup.com/downloads.html)
```